index.css

```diff
@@ -0,0 +1,43 @@
+body{
+    background-image: url("/img/jess-bailey-l3N9Q27zULw-unsplash.jpg");
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.main-container{
+    text-align: center;
+}
+
+.main-heading{
+    font-size: 80px;
+    font-weight: bold;
+    color: orangered;
+}
+
+.main-info p{
+    font-size: 25px;
+}
+
+.btn-Explore{
+    width: 130px;
+    height: 60px;
+    border: none;
+    font-size: 20px;
+    color: black;
```

Comment:
`outline: none`
You may add this here as well.

Reply:
Solved !

Reply:
You should not remove the outline it's there for accessibility reasons.